pipeline/scripts/ci/upstream_cli.py

```diff
@@ -0,0 +1,89 @@
+"""
+This script provides the upstream latest rpm repo path and container image.
```

Comment:
The module does not emit a record... 

- there is no print
- (or) file output giving the intended information.